README.md

```diff
@@ -26,13 +28,20 @@ Notebooks have been removed from the main repo, but with this exploratory and ed
 
 ### To read
 
-You can clone this repo by typing on your console:
+* You can clone this repo by typing on your console:
 
 ```
 $ git clone https://github.com/okfn-brasil/notebooks.git
 ```
 
-Alternatively you can download all the notebooks clicking the green "Clone or download" button on your right and choosing "Download ZIP". 
+* Alternatively you can download all the notebooks clicking the green "Clone or download" button on your right and choosing "Download ZIP".
+
+* Alternatively you can launch in [binder](https://mybinder.org/v2/gh/okfn-brasil/notebooks/master?filepath=notebooks).
+> This provides a simple and disposable environment to run and visualize notebooks. Don't use that as your development environment and definitely don't add your credentials on this environment.
+
+* Alternatively you can visualize the notebooks in `read only` mode using a public version of nbviewer:
+
+  https://nbviewer.jupyter.org/github/okfn-brasil/notebooks/tree/master/
```

Comment:
I believe it is better to provide this link in a markdown way something like:


Alternatively you can visualize the notebooks in `read only` mode [using a public version of nbviewer](https://nbviewer.jupyter.org/github/okfn-brasil/notebooks/tree/master/).


What do you think?

Reply:
It's a good idea.

Reply:
Done.